phpunit.xml

```diff
@@ -1,22 +1,16 @@
-<phpunit
-	bootstrap="tests/bootstrap.php"
-	colors="true"
-	backupGlobals="false">
-
-	<testsuites>
-		<testsuite name="tests">
-			<directory suffix=".php">./tests</directory>
-		</testsuite>
-	</testsuites>
-
-	<filter>
-		<whitelist processUncoveredFilesFromWhitelist="true">
-			<directory suffix=".php">src</directory>
-		</whitelist>
-	</filter>
-
-	<listeners>
-		<listener class="\Mockery\Adapter\Phpunit\TestListener"></listener>
```

Comment:
Listener was deleted because:
PHPUnit 5.7 and Mockery 1.3 had conflict generated by https://github.com/mockery/mockery/blob/1.3.3/library/Mockery/Adapter/Phpunit/Legacy/TestListenerTrait.php#L26